apps/studio/components/layouts/SQLEditorLayout/SQLEditorNavV2/SQLEditorTreeViewItem.tsx

```diff
@@ -294,6 +294,16 @@ export const SQLEditorTreeViewItem = ({
                   Move query
                 </ContextMenuItem_Shadcn_>
               )}
+              {onSelectDuplicate !== undefined && isOwner && (
```

Comment:
Just to check, is there a reason the user needs to be an owner to duplicate a query? I think it would be useful to be able to duplicate other team members queries to change variables in them etc